saleor/graphql/order/mutations/draft_order_create.py

```diff
@@ -279,13 +281,14 @@ def _save_addresses(info, instance: models.Order, cleaned_input):
     @staticmethod
     def _save_lines(info, instance, lines_data):
         if lines_data:
+            site = load_site(info.context)
```

Comment:
We are fetching the sites two times in this mutation, IMO, we should do it once in `perform_mutation` and pass to all methods that require `site` object.

Reply:
I think it calls the DB only once, as there is a data loader underneath. Then we should be able to call `load_*` as many times as we want within a single request without worrying about DB. This would look a bit better than passing the site to other mutation methods. But if the queries are constant, that would be the preferred solution.

Reply:
Both of you are right :smile: Other methods should get `site` as param as this is more testable. And dataloader ensures that the DB is only hit once - the answer is cached for future use.